users/tests.py

```diff
@@ -1 +1,48 @@
-# Create your tests here.
+from rest_framework.test import APIClient
+from rest_framework.test import APITestCase
+from users.models import OneTimePassword, User
+from plio.settings import API_APPLICATION_NAME
+from oauth2_provider.models import Application
+from django.core.management import call_command
+from django.urls import reverse
+from rest_framework import status
+
+
+class BaseTestCase(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        call_command("pliosuperuser")
+        self.super_user = User.objects.first()
+        Application.objects.create(
+            name=API_APPLICATION_NAME,
+            redirect_uris="",
+            user=self.super_user,
+            client_type=Application.CLIENT_CONFIDENTIAL,
+            authorization_grant_type=Application.GRANT_AUTHORIZATION_CODE,
+        )
+
+
+class UserAuthenticationTestCase(BaseTestCase):
```

Comment:
this tests only the OTP part right? Not the G-auth one? Would it be better to clarify that in the name?

Reply:
+ one line description